modules/monitoring-determining-why-prometheus-is-consuming-disk-space.adoc

```diff
@@ -42,3 +45,10 @@ topk(10,count by (job)({__name__=~".+"}))
 ** *If the metrics relate to a user-defined project*, review the metrics key-value pairs assigned to your workload. These are implemented through Prometheus client libraries at the application level. Try to limit the number of unbound attributes referenced in your labels.
 
 ** *If the metrics relate to a core {product-title} project*, create a Red Hat support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
+
+. Check the TSDB status in the Prometheus UI.
+.. In the *Administrator* perspective, navigate to *Networking* -> *Routes*.
+.. Select the `openshift-monitoring` project in the *Project:* list.
+.. Select the URL in the `prometheus-k8s` row to open the login page for the Prometheus UI.
+.. Choose *Log in with OpenShift* to log in using your OpenShift credentials.
```

Comment:
Should "OpenShift" be {product-title}?

Reply:
It could be `{product-title}`, but I think to stay consistent with the same steps [here](https://docs.openshift.com/container-platform/4.7/monitoring/accessing-third-party-uis.html#accessing-third-party-uis-using-the-web-console_accessing-third-party-uis), just "OpenShift" is okay. Especially since the log in prompt says "with OpenShift" and not "with OpenShift Container Platform."